install/deploy/networks/holesky.yml

```diff
@@ -10,4 +10,5 @@ networkResources:
   txWatchUrl: https://holesky.etherscan.io/tx
   flashbotsProtectUrl: https://rpc-holesky.flashbots.net/
 hyperdriveResources:
-  nodeSetApiUrl: https://nodeset.io/api
\ No newline at end of file
+  nodeSetApiUrl: https://nodeset.io/api
```

Comment:
is this the correct endpoint?

Reply:
Yes, the public testnet uses the same nodeset.io endpoint as mainnet.